cmd/start.go

```diff
@@ -18,14 +18,16 @@ import (
 	"path/filepath"
 
 	homedir "github.com/mitchellh/go-homedir"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"github.com/wpengine/lostromos/crwatcher"
 	"github.com/wpengine/lostromos/tmplctlr"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/wait"
+	wait "k8s.io/apimachinery/pkg/util/wait"
```

Comment:
Why are we renaming this package to wait when it's name is already wait?

Reply:
This was probably gofmt, I didn't change this intentionally.  I'll remove the rename.